river/tree/mondrian/mondrian_tree_regressor.py

```diff
@@ -0,0 +1,433 @@
+from __future__ import annotations
+
+import math
+import sys
+
+from river import base, utils
+from river.tree.mondrian.mondrian_tree import MondrianTree
+from river.tree.mondrian.mondrian_tree_nodes import (
+    MondrianBranchRegressor,
+    MondrianLeafRegressor,
+    MondrianNodeRegressor,
+)
+
+
+class MondrianTreeRegressor(MondrianTree, base.Regressor):
+    """Mondrian Tree Regressor.
+
+    Parameters
+    ----------
+    step
+        Step of the tree.
+    use_aggregation
+        Whether to use aggregation weighting techniques or not.
+    split_pure
+        Whether the tree should split pure leafs during training or not.
```

Comment:
Same here, maybe it needs to be removed. I don't know how to measure when a node is pure in regression. When the variance is zero?

Are you aware of that @kenzabenjelloun?

Reply:
Hi ! Thank you for taking a look at the code. 
I think it can be removed as it is not really used anyways.